pegasus/forms/hoc_survey_prize_2015.rb

```diff
@@ -0,0 +1,27 @@
+class HocSurveyPrize2015
+
+  def self.normalize(data)
+    result = {}
+
+    result[:email_s] = Poste.decrypt(data[:code_s])
+    result[:prize_choice_s] = required stripped data[:prize_choice_s]
+
+    result
+  end
+
+  def self.process(data)
+    {}.tap do |results|
+      results['prize_code_s'] = claim_prize_code(data['prize_choice_s'], data['email_s'], 'Hoc2015');
+    end
+  end
+
+  def self.receipt(data)
+
+    puts "Generating receipt!"
+    puts "data: #{data.inspect}"
+
+    suffix = data['prize_choice_s'].partition('.').first.downcase
+    suffix = 'msft' if suffix == 'microsoft'
+    "12-18-hoc-organizer-survey-#{suffix}"
```

Comment:
Good question. No prize will be assigned. See https://github.com/code-dot-org/code-dot-org/blob/71ae750f6a5e5d9719379fa42457750d61426478/pegasus/helpers/prize_helpers.rb#L9-L9

However I just realized we don't have a receipt email for the no prize case. Checking with PM